nginx-directory/files/index.html

```diff
@@ -1,20 +1,122 @@
 <html>
-  <head>
-    <style>
+
+<head>
+  <style>
+    *,
+    *:before,
+    *:after {
+      box-sizing: border-box;
+    }
+
+    body {
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
+      margin: 0;
+      padding: 24px;
+      color: #424242;
```

Comment:
could we consider using CSS variables to make the theme easier to grok / modify? could be a way to do the light/dark theme too.